modules/account-lib/src/coin/hbar/transferBuilder.ts

```diff
@@ -0,0 +1,91 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics/dist/src/base';
+import Long from 'long';
+import { AccountId } from '@hashgraph/sdk';
+import { proto } from '../../../resources/hbar/protobuf/hedera';
+import { NotImplementedError, BuildTransactionError, InvalidParameterValueError } from '../baseCoin/errors';
+import { BaseKey } from '../baseCoin/iface';
+import { TransactionBuilder } from './transactionBuilder';
+import { Transaction } from './transaction';
+import { isValidAddress, isValidAmount } from './utils';
+
+export class TransferBuilder extends TransactionBuilder {
+  private _txBodyData: proto.CryptoTransferTransactionBody;
+  private _toAddress: string;
+  private _amount: string;
+
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+    this._txBody = new proto.TransactionBody();
+    this._txBody.transactionValidDuration = this._duration;
+    this._txBodyData = new proto.CryptoTransferTransactionBody();
+    this._txBody.cryptoTransfer = this._txBodyData;
+  }
+
+  /** @inheritdoc */
+  protected async buildImplementation(): Promise<Transaction> {
+    this._txBodyData.transfers = this.buildTransferData();
+    // TODO: Add signature to the tx body
+    return super.buildImplementation();
+  }
+
+  private buildTransferData(): proto.ITransferList {
+    return { accountAmounts: [{ accountID: this.buildRecipientData(), amount: Long.fromString(this._amount) }] };
+  }
+
+  private buildRecipientData(): proto.AccountID {
+    const accountData = new AccountId(this._toAddress);

```

Comment:
we should review this approach.
We are coupling an sdk dependency just to build an `proto.AccountId`

Reply:
This is a way to take any address since it may take the format `'<shard>.<realm>.<account>'` or `'<account>'`